### ISSUE TITLE
Auswahl Chart Bibliothek

### ISSUE THREAD

**Issue by jansthh** [livingmydesign/hdstories#74]
- Zukünftig wäre es genial, die Geb.-Daten verschiedener Menschen abzuspeichern, so dass ich immer mal wieder Stories für sie schnell erzeugen kann.

- Habe mich selber öfter dabei erwischt

- ist auch genial für Geburtstage oder wenn es jemandem mal schlecht geht

- ich muss mir nochmal was Visuelles überlegen, aber für dich schon mal technisch als Hintergrundinfo 

- könnte mir ein Dropdown (ähnlich wie bei meinem Chart Rechner) bei der Story Auswahl vorstellen

- je nachdem, wen man ausgewählt hat, wird für den die Story erstellt

![image](https://github.com/livingmydesign/hdstories/assets/159889907/0f9dfd08-480d-4773-b767-4913ebae9e67)

**Comment by livingmydesign** [comment]
@jansthh ja, prinzipiell machbar. aktuell werden diese daten einfach noch nicht gespeichert.
